src/adapters/refract/getDescription.coffee

```diff
@@ -0,0 +1,10 @@
+_ = require('./helper')
+markdown = require('../markdown')
+
+module.exports = (element) ->
+  copyElement = _(element).copy().first()
+
+  raw = _.fixNewLines(_.content(copyElement) or '')
+  html = _.fixNewLines(if raw then markdown.toHtmlSync(raw) else '')
```

Comment:
Not related to this PR, but I was curious about how to make async markdown-html transformation and I realised something is terribly wrong here:

![image](https://cloud.githubusercontent.com/assets/1416224/11305035/fe04eb2c-8faa-11e5-8a56-bd591b0e1e67.png)

Please do not provide async functions for sync functions. It has no sense.